runner/entrypoint.sh

```diff
@@ -56,7 +59,14 @@ sudo chown -R runner:docker /runner
 mv /runnertmp/* /runner/
 
 cd /runner
-./config.sh --unattended --replace --name "${RUNNER_NAME}" --url "${GITHUB_URL}${ATTACH}" --token "${RUNNER_TOKEN}" ${RUNNER_GROUP_ARG} ${LABEL_ARG} ${WORKDIR_ARG}
+./config.sh --unattended --replace \
+  --name "${RUNNER_NAME}" \
+  --url "${GITHUB_URL}${ATTACH}" \
+  --token "${RUNNER_TOKEN}" \
+  "${RUNNER_GROUP_ARG}" "${RUNNER_GROUP_ARG_VALUE}" \
```

Comment:
Does entrypoint.sh or config.sh complain when `RUNNER_GROUP_ARG_VALUE` is not defined?

If not, I guess we can just rewrite these like:

```suggestion
  --runnergroup "${RUNNER_GROUP}" \
```

Reply:
Tested it, it also works since .config uses defaults on empty arguments.
Groups get assigned to default.
Also default tags.
Workdir seems also to be default.
Changed it accordingly 👍🏽